definitions/infra-ibmmq_queue/golden_metrics.yml

```diff
@@ -1,3 +1,9 @@
+manager:
+  title: Queue Manager
+  unit: COUNT
+  query:
+    select: latest(qmgr) AS 'Queue Manager'
```

Comment:
Is qmgr a metric from ibm integration? or you are expecting the tag defined above here?